src/ServiceFabric/ServiceFabric/ChangeLog.md

```diff
@@ -21,7 +21,9 @@
 
 ## Version 3.1.1
 * Added support for Windows 2022 server vm image. 
-    - This enables cluster operations with Windows 2022 server vm image 
+    - This enables cluster operations with Windows 2022 server vm image
+
+*Added new cmdlet `Add-AzServiceFabricManagedNetworkSecurityRule` to update network security rules in cluster resource
```

Comment:
this should be moved under the upcoming release section. Also should be a space after the asterisk. as well instead of `cluster resource` should say `managed cluster resource`

Reply:
updated